Build/install.cake

```diff
@@ -247,6 +247,19 @@ Task("Create-Https-Binding").Does(() =>
             .SetCertificateHash(certificate.GetCertHash())
             .SetCertificateStoreName("My")
     );
+    var computerName = "orckrefapp-ar1.orckestra.local";
+    AddBinding(localSiteName,
+        IISBindings.Http
+        .SetHostName(computerName)
+        .SetIpAddress("*")
+    );
+    AddBinding(localSiteName, 
+        IISBindings.Https
+            .SetHostName(computerName)
+            .SetIpAddress("*")
+            .SetCertificateHash(certificate.GetCertHash())
+            .SetCertificateStoreName("My")
+    );
```

Comment:
do we need this changes in Build script?

Reply:
Oops ! this is my local build setup, It got added by mistake, will remove this